src/app/RecordingMetadata/RecordingLabelFields.tsx

```diff
@@ -187,9 +194,31 @@ export const RecordingLabelFields: React.FunctionComponent<RecordingLabelFieldsP
       </Button>
       {props.isUploadable && (
         <>
-          <Button aria-label="Upload Labels" onClick={openLabelFileBrowse} variant="link" icon={<UploadIcon />}>
-            Upload Labels
-          </Button>
+          <Popover
+            isVisible={!!invalidUploads.length}
+            aria-label="uploading warning"
+            alertSeverityVariant="danger"
+            headerContent="Invalid Selection"
+            headerComponent="h1"
+            shouldClose={closeWarningPopover}
+            headerIcon={<ExclamationCircleIcon />}
+            bodyContent={
+              <>
+                <Text component="h4">The following files did not contain valid recording metadata:</Text>
```

Comment:
The only thing I would consider is that making files -> `file(s)` because there might be only 1 file. But otherwise looks good!

Reply:
Got it! Updated now :D

Reply:
May as well dynamically set the text to singular or plural depending on list size then ;-)

Reply:
Nice, I see Thuan just did that haha.

Gonna be fun when we get around to localizing these strings!

Reply:
Hoping its not gonna be a nightmare XD

Reply:
#625